src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,39 @@
 package core.basesyntax.service;
 
+import core.basesyntax.custom.exceptions.AgeNotValidException;
+import core.basesyntax.custom.exceptions.LoginNotValidException;
+import core.basesyntax.custom.exceptions.PasswordNotValidException;
+import core.basesyntax.custom.exceptions.UserNotValidException;
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 130;
```

Comment:
Developer should implement the task. Developer **should not** invent the task.
Watch this video https://www.youtube.com/watch?v=Ot2eB07rjcI

Reply:
Thanks